packages/jest-types/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@jest/types",
+  "version": "24.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git",
+    "directory": "packages/jest-types"
+  },
+  "engines": {
+    "node": ">= 6"
+  },
+  "license": "MIT",
+  "main": "build/index.d.ts",
```

Comment:
dummy `main` so `eslint-plugin-import` is happy

Reply:
There will still be a `build/index.js` generated for this package thought, can't we use that like in the other packages although there may be nothing useful in it?

Reply:
yeah, it's still generated. just an empty file with the copyright header and a `'use strict';`

Reply:
so can switch

Reply:
Hmm, it's actually more - the import isn't stripped:

```js
'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});
exports.TestResult = exports.Console = exports.Config = void 0;

var Config = _interopRequireWildcard(require('./Config'));

exports.Config = Config;

var Console = _interopRequireWildcard(require('./Console'));

exports.Console = Console;

var TestResult = _interopRequireWildcard(require('./TestResult'));

exports.TestResult = TestResult;

function _interopRequireWildcard(obj) {
  if (obj && obj.__esModule) {
    return obj;
  } else {
    var newObj = {};
    if (obj != null) {
      for (var key in obj) {
        if (Object.prototype.hasOwnProperty.call(obj, key)) {
          var desc =
            Object.defineProperty && Object.getOwnPropertyDescriptor
              ? Object.getOwnPropertyDescriptor(obj, key)
              : {};
          if (desc.get || desc.set) {
            Object.defineProperty(newObj, key, desc);
          } else {
            newObj[key] = obj[key];
          }
        }
      }
    }
    newObj.default = obj;
    return newObj;
  }
}
```

not sure if it matters

Reply:
Not stripped because Babel can't infer that the identifiers are just types in this case. Doesn't matter I'd say, in the file where you import them, the will be stripped because they are used as `TypeReference`s etc